opengrok-indexer/src/main/resources/analysis/perl/PerlXref.lex

```diff
@@ -122,6 +103,66 @@ import org.opengrok.indexer.web.HtmlConsts;
     }
 
     protected void skipLink(String s, Pattern p) { /* noop */ }
+
+    /**
```

Comment:
where are below functions used?

Reply:
They’re called by the abstract `PerlLexer` extended by the Perl JFlex xref and symtok classes. Formerly the values were injected through the helper delegate constructor but now can be mediated through method overrides.